geotrek/api/v2/serializers.py

```diff
@@ -1069,6 +1074,9 @@ def get_accessibility(self, obj):
         def get_cities(self, obj):
             return [city.code for city in obj.published_cities]
 
+        def get_equipment(self, obj):
+            return get_translation_or_dict('equipment', self, obj)
```

Comment:
idem

Reply:
This is not for the same model it is for outdoor course (we should translate it)

Reply:
- Fix translation equipment api v2 courses